hw2/05-form.html

```diff
@@ -7,13 +7,82 @@
     <title>05 - Form</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="05-form.js"></script>
   </head>
-  <body>
-    <main>
-      <form>
+  <body class="bg-dark">
+    <main class="row justify-content-center">
+      <form id="userForm" class="bg-light mt-5 w-75 p-3">
         <h1>Exercise 05 - Form</h1>
+        <label for="name" class="fw-bold">Full Name:</label> <br />
+        <input type="text" id="name" class="w-100" />
+        <label for="email" class="fw-bold">Email:</label> <br />
+        <input type="email" id="email" class="w-100" />
+        <label for="regStatus" class="fw-bold">Registration Status:</label>
+        <br />
+        <select name="regStatus" id="regStatus" class="w-100">
+          <option value="" class="fw-bold">Choose an Option</option>
+          <option value="registered">Registered</option>
+          <option value="pending">Pending</option>
+          <option value="waitlist">Waitlisted</option>
+          <option value="notRegistered">Not Registered</option>
+          <option value="other">Other</option>
+        </select>
+        <fieldset>
+          <legend class="fw-bold">
+            Have you taken any of the following courses?
+          </legend>
+          <div>
+            <input type="checkbox" id="progLangs" name="progLangs" />
+            <label for="progLangs">Programming Languages</label>
+          </div>
+          <div>
+            <input type="checkbox" id="os" name="os" />
+            <label for="os">Operating Systems</label>
+          </div>
+          <div>
+            <input type="checkbox" id="fullstack" name="fullstack" />
+            <label for="fullstack">Full Stack Web Development</label>
+          </div>
+        </fieldset>
+        <label for="also" class="fw-bold">Anything else?</label> <br />
+        <input type="text" id="also" class="w-100 h-auto" />
```

Comment:
I think this is supposed to be a <textarea> instead of 'input' with type="text".

And you can add the attribute rows="3" to the textarea element to make it look like it accepts a block of text instead of a line of text.

There's some more information about textareas [here](https://getbootstrap.com/docs/4.0/components/forms/#form-controls).